packages/gatsby-theme-carbon/src/components/Caption/caption.scss

```diff
@@ -2,7 +2,17 @@
   @include carbon--type-style('body-short-01');
   padding-top: $spacing-03;
   margin-bottom: $spacing-06;
-  margin-top: -$spacing-06; // to account for default spacing below images
+}
+
+// Caption after images
+.gatsby-resp-image-wrapper + .#{$prefix}--caption,
```

Comment:
Do we need this if we're targeting the `img` as well?

Reply:
Yes, because the actual image tag is nested inside so we need to target this for the sibling selector to work